### PR TITLE
Topic 01 exercise 01

### PR DESCRIPTION
**Pull request by fyunes**
Exercise 1, 2 and 3 of JS.

### CODE REVIEW

File: Topic-01-JSExercise-01/script.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    httpfetch.send();
+    return promise;
+}
+function pHandler() {
+    let configObject = {
+        method: 'GET',
+        url: 'http://api.icndb.com/jokes/random',
+    };
+    let promise = fetcher(configObject);
+    promise.then(function (data) {
```
Comment (DemianMoschin-GLB) [inline Topic-01-JSExercise-01/script.js]: es6 sintax for functions....

**Comment by DemianMoschin-GLB** [comment]
Looks great! keep up!

**Comment by DemianMoschin-GLB** [comment]
Please answer pending comments

File: Topic-01-JSExercise-01/script.js (hunk 1 above)
Comment (fyunes) [reply Topic-01-JSExercise-01/script.js]: fixed

**Pull request merged**